### DISCUSSION TITLE
Uninstall Homebrew: lose packages

### DISCUSSION THREAD

**Discussion by DutchPete** [Homebrew/discussions#3690] [Everyday usage]
When you uninstall Homebrew, is it still true that all the apps you installed with Hb will also be uninstalled?

If so, is there a way to avoid that?

**Comment by carlocab** [comment]
> When you uninstall Homebrew, is it still true that all the apps you installed with Hb will also be uninstalled?

Yes, if you use the uninstall script.

> If so, is there a way to avoid that?

Don't use the uninstall script, I guess. But you'll need to work out on your own what you want to happen when you uninstall Homebrew but keep the installed packages.

**Reply by DutchPete** [reply]
Sorry, I don't know if I posted my reply to you correctly. I hope this one is OK.
My questions about your reply were:

What do you mean by “what you want to happen”?
What command do I need to run?

**Reply by carlocab** [reply]
By "what you want to happen" I am referring to what you mean by "uninstall Homebrew but keep Homebrew-installed packages installed".

As for commands you need to run, that again depends on what exactly it is you're looking to do. I don't know what that is, so I don't know what those commands are.

**Reply by DutchPete** [reply]
> As for commands you need to run, that again depends on what exactly it is you're looking to do. I don't know what that is, so I don't know what those commands are.

I am on MacOS Mojave 10.14.6 and each time I do `brew upgrade` it takes a long time (1 hour+) to run the `make` process. Today, it got stuck on `cmake` because after 3-4 hours it had not moved on. I am now sick and tired of Homebrew and just want to uninstall it but retain the apps I installed with it.

**Comment by DutchPete** [comment]
Can anyone tell me specifically how to uninstall Homebrew and keep the pckages installed by it?
